### PR TITLE
Update python.yaml - Add Kaitaistruct python runtime

### PR DESCRIPTION
**Pull request by crytpy**
added kaitaistruct to rosdep

Please add the following dependency to the rosdep database.

## Package name:

python3-kaitaistruct

## Package Upstream Source:
https://pypi.org/project/kaitaistruct/

## Purpose of using this:

This dependency is used as the Kaitai Struct declarative parser generator for binary data: runtime library for Python. It is used as the runtime for python kaitaistruct generated binary parsers.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - [REQUIRED](https://pypi.org/project/kaitaistruct/)
- Ubuntu: https://packages.ubuntu.com/
  - [REQUIRED](https://pypi.org/project/kaitaistruct/)

# Please Add This Package to be indexed in the rosdistro.

GALACTIC

# The source is here:
https://github.com/kaitai-io/kaitai_struct_python_runtime

# Checks
 - [ YES] All packages have a declared license in the package.xml
 - [YES ] This repository has a LICENSE file
 - [YES ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Review by sloretz — CHANGES_REQUESTED** [review]
On the `Files changed` page the checker shows the package is available on fedora, opensuse, and debian. Mind updating the PR with those entries?

**Comment by crytpy** [comment]
Thanks @sloretz for the request - changes added.

File: rosdep/python.yaml [hunk 1]
```diff
@@ -6611,6 +6611,12 @@ python3-jupyros-pip:
   ubuntu:
     pip:
       packages: [jupyros]
+python3-kaitaistruct:
+  debian: [python3-kaitaistruct]
+  fedora: [python3-kaitaistruct]
+  opensuse: [python3-kaitaistruct]
+  ubuntu:
+    pip: [kaitaistruct]
```
Comment (clalancette) [inline rosdep/python.yaml:6619]: It looks like this is also available on Ubuntu: https://packages.ubuntu.com/search?keywords=python3-kaitaistruct&searchon=names&suite=all&section=all

```suggestion
  ubuntu: [python3-kaitaistruct]
```

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by clalancette** [comment]
There's been no update to the latest comment in over 2 weeks, so I'm going to close this out.  If you are still interested in adding this key, please reopen and respond to the latest comment.  Thank you.

**Pull request closed without merging**